main.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+app = FastAPI()
+
+class TemperatureData(BaseModel):
+    city: str
+    temperature: float
+
+# Словник для зберігання температури для міст
+temperature_db = {}
+
+@app.get("/")
+def read_root():
+    return {"message": "Welcome to the City Temperature Management API"}
+
+@app.get("/temperature/{city}")
+def get_temperature(city: str):
+    temperature = temperature_db.get(city)
+    if temperature is not None:
+        return {"city": city, "temperature": temperature}
+    return {"city": city, "temperature": "Data not available"}
```

Comment:
Consider using HTTP status codes to indicate the success or failure of the request. For instance, when data is not available, you might want to return a 404 status code to indicate that the resource was not found.